src/static/containers/ZeroCard.jsx

```diff
@@ -79,6 +80,31 @@ class ZeroCard extends Component {
         </Row>
     ) : null;
 
+    renderIcons = () => {
+        const {
+            card: {
+                published,
+                card_stats: {
+                    co2_reduction,
+                    waste_reduction,
+                    water_use_reduction,
+                },
+            },
+        } = this.props;
+        const waste = waste_reduction ? <FaTrash color="green" /> : <FaTrash color="red" />;;
+        const co = co2_reduction ? <FaCloud color="green" /> : <FaCloud color="red" />;;
+        const water = water_use_reduction ? <FaTint color="green" /> : <FaTint color="red" />;
+        const approved = published ? <FaCheck color="green" /> : <FaClose color="red" />;
```

Comment:
We need more icons:
- description not empty
- image uploaded
- cost/waste-reduction/difficulty scores not empty